Fix workbooks deployment permissions documentation (#1629)
microsoft/finops-toolkit: docs-mslearn/toolkit/workbooks/finops-workbooks-overview.md

1. To deploy and use the workbook, confirm you have the following least-privileged roles:

>    - **Contributor** or a role with both `Microsoft.Resources/deployments/validate/action` and `Microsoft.Resources/deployments/write` permissions is required for ARM template deployments.

Reviewer: ```suggestion
```

   - **Workbook Contributor** allows you to save the workbook after deployment.

>    - **Reader** to view all of the workbook tabs.

Reviewer 2: I was testing the deployment for v0.10, and for the deployment it needed Contributor or a role with Microsoft.Resources/deployments/validate/action and Microsoft.Resources/deployments/write, plus Workbook Contributor, and Reader.

   > [!NOTE]
   > If you only have read access, you can still import your workbook directly into Azure Monitor. You will not be able to save it, but you can view all tabs. You can download the workbook JSON files from the FinOps workbooks download in the [latest release](https://aka.ms/ftk/latest).

2. Deploy the **finops-workbooks** template using one of the following methods:

>    **Option 1:** Deploy using ARM template (requires Contributor access)

Reviewer 2: ```suggestion
   **Option 1:** Deploy using ARM template (requires Microsoft.Resources/deployments permissions)
```

   <a href="https://portal.azure.cn/#create/Microsoft.Template/uri/https%3A%2F%2Fmicrosoft.github.io%2Ffinops-toolkit%2Fdeploy%2Ffinops-workbooks-latest.json/createUIDefinitionUri/https%3A%2F%2Fmicrosoft.github.io%2Ffinops-toolkit%2Fdeploy%2Ffinops-workbooks-latest.ui.json"><img alt="Deploy To Azure China" src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazurechina.svg?sanitize=true" /></a>
   -->

   **Option 2:** Import JSON files directly (works with Reader access)
   

>    1. Download the workbook files from the [FinOps toolkit GitHub repository](https://github.com/microsoft/finops-toolkit/tree/main/src/workbooks)

Reviewer: ```suggestion
   1. Download FinOps workbooks from the [latest release](https://aka.ms/ftk/latest).
```

   2. Navigate to [Azure Monitor Workbooks](https://portal.azure.com/#view/Microsoft_Azure_Monitoring/AzureMonitoringBrowseBlade/~/workbooks) in the Azure portal

>    3. Click on "+ New" and select "Import from file"

Reviewer: ```suggestion
   3. Click on **+ New** and select **Advanced editory**
```

>    4. Upload the downloaded .workbook file

Reviewer: ```suggestion
   4. Copy the text from the desired workbook.json from the downloaded ZIP file, paste it into the editor, and select **Apply**.
```

>    5. Click "Apply" to view the workbook

Reviewer: ```suggestion
   5. Select **Done editing** to view the workbook.
   6. Repeat steps 3-5 for each workbook.
```
